index.js

```diff
@@ -15,6 +15,8 @@ class Seeder {
   }
 
   async run(path = process.cwd()) {
+    // fail only if env variable equals "true"
+    const shouldFailEarly = !!(process.env.FAIL_EARLY === 'true')
```

Comment:
```suggestion
    const shouldFailEarly = process.env.FAIL_EARLY === 'true'
```

That statement already returns a boolean so you don't need to cast it